sources/index.js

```diff
@@ -17,6 +20,9 @@ function loadAnimation(container, animation){
         container: container,
         path: animation,
         rendererSettings: {
+            rendererSettings: {
+                preserveAspectRatio: 'xMidYMid slice'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/croqo/oc-lottie-plugin/pullRequest?prid=8474265)
```suggestion
                preserveAspectRatio: "xMidYMid slice"
```